src/scenes/Hare.js

```diff
@@ -141,27 +177,41 @@ function Hare (scene, hareCount) {
   tween3.chain(tween1);
   // tween4.chain(tween1);
 
+  var waitForHawksToFlyAway = true;
   var eating_pace = 20;
   var eating_paceCntr = eating_pace;
 
   function update () {
 
-    //TODO: this should really be real-time-based, not loop based
-    //TODO: it should also be part of a behavior model so these can be tuned the behavior models here
-    //if(eating_paceCntr-- == 0)
     {
       eating_paceCntr = eating_pace; 
       var deltaDistance = 500;
-       findRemoveIfNear(hareMesh.position, deltaDistance);
+      findRemoveIfNear(hareMesh.position, deltaDistance);
     }
-    if(movingToTree)
-    {
-      var h_x = parseInt(hareMesh.position.x.toFixed());
-      var t_x = parseInt(treePos.x.toFixed());
-      console.log("hare x: tree x " + h_x + ":"+ t_x);
-      if(h_x == t_x)
-       //&&(hareMesh.position.y == treePos.y))
-        pause();
+
+    if(movingToTree) {
+      var id = getHareID(hareMesh);
+      var distance = getDistance(hareMesh.position, treePos).toFixed();
```

Comment:
`toFixed()` isn't required here, the next line parses the value back to an Integer.